ephemeral-utils.lua

```diff
@@ -0,0 +1,26 @@
+local _Utils = {}
```

Comment:
`_M` convention comes from the earlier 5.1 module API: https://www.lua.org/manual/5.1/manual.html#5.3, just FYI